src/products/queries.ts

```diff
@@ -176,6 +177,19 @@ export const useProductListQuery = makeQuery<ProductList, ProductListVariables>(
   productListQuery
 );
 
+const productCountQuery = gql`
+  query ProductCount($filter: ProductFilterInput) {
```

Comment:
Good point! Changing

Reply:
One thing, I decided to leave it named `useProductCountQuery`, since `all` suggests it's always all products, whereas these may just as well be filtered products. Anyway both of these were used only once so it was a small change